azure-pipelines.yml

```diff
@@ -0,0 +1,120 @@
+      - task: SonarQubePrepare@4
+        displayName: Prepare analysis on SonarQube
+        inputs:
+          SonarQube: $(sonarQubeServiceConnection)
+          projectKey: nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk:dotnet
+          projectName: Rabobank OmniKassa - .NET SDK
+          extraProperties: sonar.exclusions=samples/**
+      - task: VSBuild@1
+        displayName: Build Omnikassa SDK
+        inputs:
+          solution: OmniKassa.sln
+          msbuildArgs: /p:DeployOnBuild=true /p:WebPublishMethod=Package /p:PackageAsSingleFile=true /p:SkipInvalidConfigurations=true /p:DesktopBuildPackageLocation="$(build.artifactstagingdirectory)\WebApp.zip" /p:DeployIisAppPath="Default Web Site"
+          platform: $(BuildPlatform)
+          configuration: $(BuildConfiguration)
+      - task: VSTest@2
+        displayName: Test Assemblies
+        inputs:
+          testAssemblyVer2: >-
+            **\$(BuildConfiguration)\**\*test*.dll
+
+            !**\obj\**
+          codeCoverageEnabled: true
+          platform: $(BuildPlatform)
+          configuration: $(BuildConfiguration)
+      - task: SonarQubeAnalyze@4
+        displayName: Run Code Analysis
+      - task: SonarQubePublish@4
+        displayName: Publish Quality Gate Result
+      - task: CmdLine@2
+        displayName: Set Java to Version 11
+        inputs:
+          script: >-
+            echo "##vso[task.setvariable variable=JAVA_HOME]$(JAVA_HOME_11_X64)"
+
+            echo "##vso[task.setvariable variable=PATH]$(JAVA_HOME_11_X64)\bin;$(PATH)"
+      - task: NexusIqPipelineTask@1
+        displayName: Nexus IQ policy evaluation
+        inputs:
+          nexusIqService: $(nexusServiceConnection)
+          applicationId: omnikassa-frontend-sdk-net
+          stage: Develop
+          scanTargets: src/**/*.dll
+      - task: VSBuild@1
+        displayName: Build Omnikassa SDK Samples
+        inputs:
+          solution: OmniKassa.Samples.sln
+          msbuildArgs: /p:DeployOnBuild=true /p:WebPublishMethod=Package /p:PackageAsSingleFile=true /p:SkipInvalidConfigurations=true /p:DesktopBuildPackageLocation="$(build.artifactstagingdirectory)\WebApp.zip" /p:DeployIisAppPath="Default Web Site"
+          platform: $(BuildPlatform)
+          configuration: $(BuildConfiguration)
+      - task: PostAnalysis@1
+        displayName: Post Analysis
+        inputs:
+          CredScan: true
+      - task: PublishSymbols@2
+        displayName: Publish symbols path
+        continueOnError: True
+        inputs:
+          SearchPattern: '**\bin\**\*.pdb'
+          PublishSymbols: false
+          SymbolServerType: TeamServices
+      - task: PublishBuildArtifacts@1
+        displayName: Publish Artifact
+        condition: succeededOrFailed()
+        inputs:
+          PathtoPublish: $(build.artifactstagingdirectory)
+          ArtifactName: $(BuildParameters.ArtifactName)
+          TargetPath: '\\my\share\$(Build.DefinitionName)\$(Build.BuildNumber)'
```

Comment:
what is this `\\my\share\` coming from?

Reply:
This actually comes form the visual pipeline that was created on Devops.